dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -818,7 +818,7 @@ protected void handleListNonpublishedEntries() {
     @Override
     protected void handleListUnstarredEntries() {
         try {
-            List<Workflow> workflows = workflowsApi.allPublishedWorkflows(null, null, null, null, null, false);
+            List<Workflow> workflows = workflowsApi.allPublishedWorkflows(null, null, null, null, null, false, null);
```

Comment:
Whoops, that was me that broke that.

Reply:
it happens. Arguably, the generated code should be more graceful/have overloaded methods without the optional prarameters